lib/susedistribution.pm

```diff
@@ -198,7 +198,9 @@ sub x11_start_program {
     }
     wait_still_screen(1);
     save_screenshot;
-    send_key 'ret';
+    # sometimes send_key 'ret' doesn't work because of low performance
+    # so workaround this with repeat send_key can help
+    send_key_until_needlematch 'generic-desktop', "ret", 5, 5;
```

Comment:
This will not work in general as 'generic-desktop' is only the very specific target we want to reach in case of the desktop runner test module. And also I do not think we should recheck the final screen we want to screen more than needed. As every post_run_hook of x11test based test modules would check for generic-desktop anyway we would do it twice. So besides that this idea in general does not make sense and can not work I would ensure that in the post_run_hook the desktop is only looked for if the last match was not already exactly that

Reply:
okay, then no changes here. I don't know a better to workaround issue related on openQA production server. The statistic shows only problem with low performance on server.